pywwt/core.py

```diff
@@ -191,16 +191,36 @@ def _validate_longitude(self, proposal):
     ss_orbit_color = ColorWithOpacity('darkgray', help='The color of the solar system orbit paths (:class:`str` or `tuple`)').tag(wwt='solarSystemOrbitColor')
     ss_overlays = Bool(False, help='Whether to show solar system overlays (:class: `bool`)').tag(wwt='solarSystemOverlays') # what does this mean?
     ss_scale = Int(1, help='Specifies how to scale the size of the Sun and its planets, with 1 as actual size (:class: `int`)').tag(wwt='solarSystemScale')
-    ss_stars = Bool(False, help='Whether to show solar system overlays (:class: `bool`)').tag(wwt='solarSystemOverlays')
+    ss_stars = Bool(False, help='Whether to show solar system overlays (:class: `bool`)').tag(wwt='solarSystemOverlays')'''
 
+    def set_view(self, mode):
+        """
+        Change the view mode. (...)
+
+        Parameters
+        ----------
+        mode : `str`
+            The desired mode. (default: 'sky')
+        """
+        available = ['sky', 'sun', 'mercury', 'venus', 'earth', 'moon', 'mars',
+                     'jupiter', 'saturn', 'uranus', 'neptune', 'pluto']
+        ss_mode = '3D Solar System View'
+        
+        if mode in available:
```

Comment:
users shouldn't need to worry about case sensitivity.  Perhaps `if mode.lower() in available` or ValueError prints available + 'solar_system'

Reply:
I agree!